index.html

```diff
@@ -199,80 +214,298 @@ <h2>Multi-Post <br />Stories</h2>
-        what you see and have a project you need coded, don’t hesitate to
-        contact me.
-      </p>
-
-      <div class="media">
-        <h3>LET'S CONNECT</h3>
-        <div class="socials">
-          <ul>
-            <li class="icon">
-              <a href="https://github.com/peter-mwovi/"
-                ><img src="./images/github.png" alt="github link"
-              /></a>
+    <div class="grid-1b">
+      <div class="rows">
+        <div class="photo1">
+          <img class="work1b-img" alt="work 1" src="images/image4.png" />
+        </div>
+        <div class="work1b">
+          <h2 class="work-text1b poppins">Tonic</h2>
+          <ul class="flex1">
+            <li>
+              <p class="canopy poppins">CANOPY</p>
+            </li>
+            <li>
+              <img class="dot" alt="dot" src="images/dot1.png" />
+            </li>
+            <li>
+              <p class="bed poppins">Back End Dev</p>
             </li>
-            <li class="icon">
-              <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
-                ><img src="./images/Linkedin_icon.png" alt="linked link"
-              /></a>
+            <li>
+              <p>
+                <img class="dot" alt="dot" src="images/dot1.png" />
+              </p>
+            </li>
+            <li>
+              <p class="year poppins">2015</p>
             </li>
-            <li class="icon">
-              <a href="#"><img src="./images/doits.png" alt="doit" /></a>
+          </ul>
+          <p class="desc2 poppins">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex2">
+            <li>
+              <p class="stack poppins">html</p>
             </li>
-            <li class="icon">
-              <a href="https://twitter.com/mwovi_peter"
-                ><img src="./images/twitter.png" alt="twitter link"
-              /></a>
+            <li>
+              <p class="stack poppins">css</p>
             </li>
-            <li class="icon">
-              <a href="#"><img src="./images/M.png" alt="M" /></a>
+            <li>
+              <p class="stack poppins">javascript</p>
             </li>
           </ul>
+          <nav>
+            <button type="button" class="see-project poppins">
+              See project
+            </button>
+          </nav>
         </div>
-
-        <div class="resume">
-          <button>Get My Resume</button>
+      </div>
+      <div class="rows">
+        <div class="work1b">
+          <h2 class="work-text1b poppins">Multi-Post Stories</h2>
+          <ul class="flex1">
+            <li>
+              <p class="canopy poppins">FACEBOOK</p>
+            </li>
+            <li>
+              <img class="dot" alt="dot" src="images/dot1.png" />
+            </li>
+            <li>
+              <p class="bed poppins">Full Stack Dev</p>
+            </li>
```

Comment:
- [ ] As mentioned by previous reviewer the 2nd and 4th project images are overflowing from the container please make sure to fix it 

|figma design|your design|
|--|---|
|<img width="756" alt="Screenshot 2023-06-10 at 3 06 46 PM" src="https://github.com/peter-mwovi/portfolio_setup/assets/44798044/7cb6ed84-74c2-4938-bd96-ac5091d70110">|<img width="1087" alt="Screenshot 2023-06-10 at 3 40 13 PM" src="https://github.com/peter-mwovi/portfolio_setup/assets/44798044/33811dc1-92bb-4cef-9be2-c5e6aa13ce66">|